cms/static/cms/js/require-config.js

```diff
@@ -63,10 +63,13 @@
             "xblock": "coffee/src/xblock",
             "utility": "js/src/utility",
             "accessibility": "js/src/accessibility_tools",
-            "draggabilly": "js/vendor/draggabilly.pkgd",
             "URI": "js/vendor/URI.min",
             "ieshim": "js/src/ie_shim",
             "tooltip_manager": "js/src/tooltip_manager",
+            "modernizr": "js/vendor/afontgarde/modernizr.fontface-generatedcontent",
+            "afontgarde": "js/vendor/afontgarde/afontgarde",
+            "edxicons": "js/vendor/afontgarde/edx-icons",
```

Comment:
It is rather strange to have edx-icons live in vendor, when it is not a vendor library. I'd prefer to load it directly from the Pattern Library but that will require my other changes.